fighthealthinsurance/views.py

```diff
@@ -197,107 +202,111 @@ def post(self, request):
-                    "combined": next_step_info.combined_form,
-                    "denial_form": denial_ref_form,
-                },
-            )
-        else:
-            # If not valid take the user back.
-            return render(
-                request,
-                "categorize.html",
-                context={
-                    "post_infered_form": form,
-                    "upload_more": True,
-                },
-            )
+            return render(request, "outside_help.html", context={
+                "outside_help_details": next_step_info.outside_help_details,
+                "combined": next_step_info.combined_form,
+                "denial_form": denial_ref_form,
+            })
+
+        # If not valid take the user back.
+        return render(
+            request,
+            "categorize.html",
+            context={
+                "post_infered_form": form,
+                "upload_more": True,
+            },
+        )
 
 
 class ChooseAppeal(View):
     def post(self, request):
-        form = ChooseAppealForm(request.POST)
-        if form.is_valid():
-            appeal_info_extracted = ""
-            (appeal_fax_number, insurance_company, candidate_articles) = (
-                ChooseAppealHelper.choose_appeal(**form.cleaned_data)
-            )
-            fax_form = FaxForm(
-                initial={
-                    "denial_id": form.cleaned_data["denial_id"],
-                    "email": form.cleaned_data["email"],
-                    "semi_sekret": form.cleaned_data["semi_sekret"],
-                    "fax_phone": appeal_fax_number,
-                    "insurance_company": insurance_company,
-                }
-            )
-            # Add the possible articles for inclusion
-            if candidate_articles is not None:
-                for article in candidate_articles[0:6]:
-                    article_id = article.pmid
-                    title = article.title
-                    link = f"http://www.ncbi.nlm.nih.gov/pubmed/{article_id}"
-                    label = mark_safe(
-                        f"Include Summary* of PubMed Article <a href='{link}'>{title} -- {article_id}</a>"
-                    )
-                    fax_form.fields["pubmed_" + article_id] = forms.BooleanField(
-                        label=label, required=False, initial=True
-                    )
-
-            return render(
-                request,
-                "appeal.html",
-                context={
-                    "appeal": form.cleaned_data["appeal_text"],
-                    "user_email": form.cleaned_data["email"],
-                    "denial_id": form.cleaned_data["denial_id"],
-                    "appeal_info_extract": appeal_info_extracted,
-                    "fax_form": fax_form,
-                },
-            )
-        else:
+        form = core_forms.ChooseAppealForm(request.POST)
+
+        if not form.is_valid():
             print(form)
```

Comment:
Maybe a TODO here too to do something useful later

Reply:
I just assumed that print statements were already TODOs 😹 I could add that though